genienlp/tasks/base_dataset.py

```diff
@@ -1,11 +1,11 @@
 import os
 import zipfile
 import tarfile
+from six.moves import urllib
```

Comment:
While we're changing this, we should import the python3 version directly, instead of importing from six. We don't want six.

Reply:
fixed